app/scripts/modules/amazon/src/image/image.reader.ts

```diff
@@ -0,0 +1,49 @@
+import { IPromise, module } from 'angular';
+import { $q } from 'ngimport';
+
+import { API } from '@spinnaker/core';
+
+export interface IAmazonImage {
+  accounts: string[];
+  amis: {
+    [region: string]: string[];
+  };
+  attributes: {
+    creationDate: string;
+    virtualizationType: string;
+  };
+  imageName: string;
+  tags: {
+    [tag: string]: string;
+  };
+  tagsByImageId: {
+    [imageId: string]: IAmazonImage['tags'];
+  };
+}
+
+export class AwsImageReader {
```

Comment:
Looks like this doesn't need angular at all... how about just make it a static non-angular class while you're here?

Reply:
fine

Reply:
nope. this is still needed by aws module 😢 

https://github.com/spinnaker/deck/blob/e1a33930a6b522f9349c4ccbe1abc4aa93bb6e1f/app/scripts/modules/amazon/src/aws.module.ts#L82-L92

Reply:
De-angularized this in a new commit.